app/src/main/res/values/strings.xml

```diff
@@ -135,4 +135,7 @@
         <item>Projektierte Felsen \u2117</item>
     </string-array>
 
+    <!-- What's New dialog -->
+    <string name="whats_new">Was ist neu</string>
```

Comment:
1. There is already a `dialog` section in this file.
2. Maybe "Neuigkeiten" is the better slogan here?